Specs/DateTime/Portuguese/DateExtractor.json

```diff
@@ -34,6 +34,20 @@
   ]
 }
 ,
+{
+  "Input": "nas ultimas 3 semanas",
+  "NotSupported": "javascript",
+  "NotSupportedByDesign": "python",
+  "Results": [  ]
```

Comment:
Cause exception, so no result?

Reply:
Nope. This should never be extracted as this type.